client/package.json

```diff
@@ -234,10 +231,13 @@
 		"yamlparser": "^0.0.2"
 	},
 	"devDependencies": {
-		"@testing-library/react": "^11.2.5",
+		"@testing-library/jest-dom": "^5.9.0",
+		"@testing-library/react": "^10.0.5",
```

Comment:
Note: there were two incompatible dependencies in the file. v10 in `dependencies` and v11 in `devDependencies`. I imagine v10 would be resolved in that case? If tests pass, I'm inclined to leave this at v10.

Reply:
I actually don't know what happens if you have two different versions. I guess it depends on which is installed first?

Usually I'd go with whatever version we are already using in the repo, but we have both `@testing-library/react@^10.0.5` and `@testing-library/react@^11.2.5`. In that case I tend to go with the highest version because it may have some bugfixes not backported to the old version, but not a strong recommendation, v10 is fine as well.

Reply:
I'll bump to v11 and keep it if the tests still pass